CMakeLists.txt

```diff
@@ -34,6 +34,19 @@ size_t malloc_usable_size(const void* ptr) { return 0; }
 int main() { return 0; }
 " CONST_QUALIFIED_MALLOC_USABLE_SIZE)
 
+# older libcs might not have getentropy, e.g. it appeared in gliobc 2.25
+# so we need to fallback if we cannot compile this
+CHECK_C_SOURCE_COMPILES("
+#if __has_include(<sys/random.h>)
```

Comment:
Could you add a
```
#include <unistd.h>
```
I seem to remember some platforms include it in there.

Reply:
good call! :) done that.